packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -99,6 +99,65 @@ <h3 id="attachments-multipleFile" hidden>Attachments</h3>
   </div>
   <!-- FileUploader: end -->
 
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Input Multiple with validation</h2>
```

Comment:
Please use Title Case like in the other headings:

```suggestion
  <h2 class="docs-Heading">Input Multiple with Validation</h2>
```

Reply:
fixed